src/factory.ts

```diff
@@ -56,11 +58,13 @@ const allReleaseTypes = [
   'go',
   'go-yoshi',
   'helm',
+  'java',
```

Comment:
Do we expect folks to ever use this strategy directly if it doesn't update any files? It's possible to pair with the `extra-files`.

Reply:
I believe it makes sense to make it available, since it can handle some niche cases. where all replaces will be handled by `extra-files`.

Reply:
SGTM